### PR TITLE
Add ArangoDB health check

### PR DESCRIPTION
**Pull request by ahryshchanka**
**What this PR does / why we need it**:
This PR add health check for [ArangoDB](https://www.arangodb.com/)

**Which issue(s) this PR fixes**:
It is new project and implementation.

Please reference the issue this PR will close: #479 

**Special notes for your reviewer**:
ArangoDB recommend 3 .net clients:
[[1]](https://github.com/ra0o0f/arangoclient.net/) , last commit 2 years ago
[[2]](https://github.com/yojimbo87/ArangoDB-NET) , last commit 3 years ago
[[3]](https://github.com/Actify-Inc/arangodb-net-standard) , last commit 2 months ago
[link](https://www.arangodb.com/arangodb-drivers/) 
image

**Does this PR introduce a user-facing change?**:
Yes. New extension mentod for `IHealthChecksBuilder` and create new nuget package.

Please make sure you've completed the relevant tasks for this PR, out of the following list:

- [x] Code compiles correctly
- [x] Created/updated tests
- [x] Unit tests passing
- [x] End-to-end tests passing
- [x] Extended the documentation
- [ ] Provided sample for the feature

### CODE REVIEW

File: test/FunctionalTests/HealthChecks.ArangoDb/ArangoDbHealthCheckTests.cs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using System.Net;
+using System.Threading.Tasks;
+using HealthChecks.ArangoDb;
+using Xunit;
+
+namespace FunctionalTests.HealthChecks.ArangoDb
+{
+
```
Comment (sungam3r) [inline test/FunctionalTests/HealthChecks.ArangoDb/ArangoDbHealthCheckTests.cs:15]:
```suggestion
```

Comment (ahryshchanka) [reply test/FunctionalTests/HealthChecks.ArangoDb/ArangoDbHealthCheckTests.cs]: fixed

**Comment by unaizorrilla** [comment]
Hi @ahryshchanka 

I try to review this a soon as possible. Thanks for contribute again on this project!

**Comment by antonhryshchanka** [comment]
> Hi @ahryshchanka
> 
> I try to review this a soon as possible. Thanks for contribute again on this project!

Thank you. I'll fix conflicts today or tomorrow.

**Comment by unaizorrilla** [comment]
Hi @ahryshchanka 

Merged, a new release build is on appveyor, when finsished a new package 3.1.1-preview will be available on nuget!

**Pull request merged**